tests/phpunit/integration/api/image/beans-image-editor/createEditedImage.php

```diff
@@ -53,6 +54,9 @@ public function test_should_edit_create_and_store_image() {
 		);
 		$args                 = array( 'resize' => array( 800, false ) );
 
+		// Return the virtual filesystem's path to avoid wp_normalize_path converting its prefix from vfs::// to vfs:/.
+		Monkey\Functions\when( 'wp_normalize_path' )->returnArg();
```

Comment:
Let's move this to the `setUp()` method in the Image API's Base Test Case.  Then it's there for all of the tests.

Reply:
No problem.
I thought that might interfere with when we want wp_normalize_path to have its original functionality.